frontend/src/components/auth/Login.tsx

```diff
@@ -55,65 +47,115 @@ const Login = (): React.ReactElement => {
     setAuthenticatedUser(user);
   };
 
-  const onSignUpClick = () => {
-    navigate(SIGNUP_PAGE);
-  };
-
-  const onGoogleLoginSuccess = async (idToken: string) => {
-    const user: AuthenticatedUser = await authAPIClient.loginWithGoogle(
-      idToken,
-      loginWithGoogle,
-    );
-    setAuthenticatedUser(user);
-  };
-
   if (authenticatedUser) {
     return <Navigate replace to={HOME_PAGE} />;
   }
 
   return (
-    <div style={{ textAlign: "center" }}>
-      <h1>Login</h1>
-      <form>
-        <div>
-          <Input
-            type="email"
-            value={email}
-            onChange={(event) => setEmail(event.target.value)}
-            placeholder="username@domain.com"
-          />
-        </div>
-        <div>
-          <Input
-            type="password"
-            value={password}
-            onChange={(event) => setPassword(event.target.value)}
-            placeholder="password"
-          />
-        </div>
-        <div>
-          <Button onClick={onLogInClick}>Log In</Button>
-        </div>
-        <GoogleLogin
-          text="continue_with"
-          onSuccess={(response: CredentialResponse): void => {
-            if (response?.credential) {
-              onGoogleLoginSuccess(response.credential);
-            } else {
-              // eslint-disable-next-line no-alert
-              window.alert(response);
-            }
-          }}
-          onError={() =>
-            // eslint-disable-next-line no-alert
-            window.alert("An error occurred while authenticating with Google.")
-          }
-        />
-      </form>
-      <div>
-        <Button onClick={onSignUpClick}>Sign Up</Button>
-      </div>
-    </div>
+    <Flex
+      flexDirection="column"
+      width="100wh"
+      height="100vh"
+      justifyContent="center"
+      alignItems="center"
+    >
+      <VStack
+        justify="space-between"
+        border={{ base: "0px", md: "1px" }}
+        borderColor="#D6D6D6"
+        borderRadius="5%"
+        padding={{ base: "4% 3% 4% 3%", md: "4% 7% 4% 7%" }}
+        width={{ base: "80%", md: "40%" }}
+        height={{ base: "60%", md: "75%" }}
```

Comment:
The responsiveness seems mostly good. However, there are few issues with some screen sizes. See image below as example. I would suggest altering the height included here to deal with this.

![image](https://user-images.githubusercontent.com/63923219/216461927-ecacd505-a216-40ec-bf7e-ab2406744fd3.png)